src/MainWindow.cpp

```diff
@@ -295,7 +298,7 @@ void MainWindow::updateActions()
     _ui->actionReadExcludedDirectory->setEnabled      ( oneDirSelected && sel->isExcluded()   );
 
     bool showingTreemap = _ui->treemapView->isVisible();
-
+    _ui->actionTreemapAsSidePanel->setEnabled( showingTreemap );
```

Comment:
Please retain existing empty lines. They serve to group things logically together.